.github/workflows/update-helm-docs.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update Helm Docs
+
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
Change the rule, we don't want it to happen on every push to a branch

Reply:
ok